azkaban-common/src/main/java/azkaban/utils/Utils.java

```diff
@@ -455,4 +466,26 @@ public static long parseMemString(String strMemSize) {
 
     return sizeInKb;
   }
+
+  /**
+   * @param cronExpression: A cron expression is a string separated by white space, to provide a parser and evaluator for Quartz cron expressions.
+   * @return : org.quartz.CronExpression object.
+   */
+  public static CronExpression parseCronExpression(String cronExpression, DateTimeZone timezone) {
```

Comment:
Thanks for reminder! I just realized BasicTimeChecker class have the same logic for parsing cron Expression. I might use this util function to replace it.